pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -280,6 +282,8 @@ func (o *EditOptions) Run() error {
 			switch o.EditMode {
 			case NormalEditMode:
 				err = o.visitToPatch(infos, updatedVisitor, &results)
+			case ApplyEditMode:
+				err = o.applyEditPatch(infos, updatedInfos)
```

Comment:
do you think we can change this in a separated pr, and maybe after code freeze ?